src/chain.h

```diff
@@ -324,6 +324,11 @@ class CDiskBlockIndex : public CBlockIndex
             READWRITE(VARINT(nDataPos));
         if (nStatus & BLOCK_HAVE_UNDO)
             READWRITE(VARINT(nUndoPos));
+        uint8_t tag = 0;
```

Comment:
Yes, you're right. More precisely we need a release that resets both testnet and mainnet in order to drop this.